content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -776,7 +777,7 @@ node/mynode drained
 
```

Comment:
Above, update the `kubectl` example to point out that kubectl cannot impersonate extra and UID.